packages/vite-plugin-svelte/src/ui/inspector/Inspector.svelte

```diff
@@ -222,9 +257,11 @@
 			.join('')}`}
 		on:click={() => toggle()}
 		bind:this={toggle_el}
+		aria-label={`${enabled ? 'disable' : 'enable'} svelte-inspector`}
```

Comment:
`aria-label` won't have any effect on a div. Can this be a `button` instead?

Reply:
yes, button is very possible. thanks for noticing

Reply:
is a button now, thank you `all: unset`